_posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md

```diff
@@ -0,0 +1,128 @@
+[lesson documentation](https://github.com/swcarpentry/lesson-example)
+and
+[workshop template](https://github.com/swcarpentry/workshop-template)
+**will have a new home in 2018**.
+If you have questions or want to help with this migration,
+[Christina Koch](mailto:koch.christinal@gmail.com) is the person you should contact.
+
+During the migration,
+we will solve the divergences between
+[Software Carpentry workshop template](https://github.com/swcarpentry/workshop-template)
+and
+[Data Carpentry workshop template](https://github.com/datacarpentry/workshop-template).
+If you have questions or want to help with this migration,
+[Maneesha Sane](mailto:maneesha@software-carpentry.org) is the person you should contact.
+
+## Keyboard key visual look
+
+To improve the reader experience of the lesson,
+we will make a visual diference of keyboard keys that need to be pressed by the reader.
+We expected to merge the new CSS and documentation in the next few weeks
+and that release 2018.6 of all the lessons complain with the new visual.
+More information about the new visual is available [at this pull request](https://github.com/swcarpentry/styles/pull/165).
+Thanks to [Brandon Curtis](https://github.com/brandoncurtis) to propose the new visual.
+
+## Jekyll/Liquid include for images/figures
+
+To improve the reader experience of the lesson by providing a more uniform render of images,
+we will pursuit the proposal on [GitHub issue styles#161](https://github.com/swcarpentry/styles/issues/161)
+after we review [lessons unit test suite](https://github.com/swcarpentry/styles/blob/gh-pages/bin/lesson_check.py)
+and its use by a continuous integration platform.
+
+## Citing the templates
+
+If you want are reusing the lesson template
+and you to credit us,
+please use [Software Carpentry: Example Lesson at Zenodo](https://zenodo.org/record/838778#.WbPw1HVifCl).
+
+## Lesson release and hosting scheme
+
+[For years](http://lists.software-carpentry.org/pipermail/maintainers/2016-April/000230.html),
+we want to point readers to the last release of our lessons
+but due technical limitations on GitHub Pages
+and all the trouble that branches created for Git novice users
+(for example,
+the current branch isn't clear when visiting the lesson homepage in GitHub
+and
+maintainers not be able to change the target branch of a pull request)
+we stayed with a single `gh-pages` branch in the Git repository.
+
+[Jonah Duckles opened a issue](https://github.com/swcarpentry/lesson-example/issues/126)
+to discuss possible solutions to this issue.
+If you want to contribute with the discussion
+left your comments on the [GitHub issue](https://github.com/swcarpentry/lesson-example/issues/126).
+
+## Fully-offline-capable functionality in lesson navigation
+
+[vuw-ecs-kevin GitHub user](https://github.com/vuw-ecs-kevin)
+requested for us to improve the reader experience for those that get our lesson from [Zenodo](https://zenodo.org/),
```

Comment:
requested that we improve the readers experience, if people come to our lesson from Zenodo